### DISCUSSION TITLE
When uploading some pictures, it shows that something went wrong. It is no problem to change other pictures. How can I solve it?

### DISCUSSION THREAD

**Discussion by HoratioShaw** [Chevereto/Chevereto-Free#160] [Help needed]
When uploading some pictures, it shows that something went wrong. It is no problem to change other pictures. How can I solve it?

**Comment by rodber** [comment]
You need to debug to understand why the issue happens.

https://chevereto-free.github.io/manual/troubleshooting/debug.html
